libs/environment/src/components/network-switcher/network-switcher.tsx

```diff
@@ -137,6 +138,16 @@ export const NetworkSwitcher = () => {
             ))}
           </>
         )}
+        <div
+          className="relative flex items-center justify-between mx-2 py-2 border-t border-neutral-400 pt-2 text-sm"
+          key="propose-network-param"
+        >
+          <ExternalLink
+            href={`${VEGA_TOKEN_URL}/governance/propose/network-parameter`}
```

Comment:
suggestion (non-blocking): we are changing these URLs in future so I'd advise keeping these in a map somewhere were it will be easy to update all token URLs if the routing structure changes.